pages/profile.html

```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<link rel="stylesheet" href="./common.css" />
+		<title>Cart Page</title>
+
+		<script
+			src="https://kit.fontawesome.com/cb91afed1c.js"
+			crossorigin="anonymous"
+		></script>
+		<script>
+			function openMobileSidebar() {
+				var mobileBar = document.getElementById("mobile-sidebar");
+				mobileBar.style.display = "block";
+			}
+			function closeMobileSidebar() {
+				var mobileBar = document.getElementById("mobile-sidebar");
+				mobileBar.style.display = "none";
+			}
+		</script>
```

Comment:
Just a suggestion keep separated file for js

Reply:
I will implement in react js. I used it here to just make the functionality work.